.changeset/red-tables-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-react': patch
+---
+
+Add pop over on the `EntityRefLink` component. It shows a more details about the associated entity.
```

Comment:
This is super awesome! Many thanks! 

Would it be possible to include an example? I know you have a StoryBook example maybe a link to that?

Reply:
I believe it will appear here when it is merged. https://backstage.io/storybook/?path=/story/catalog-peekaheadpopover--default

Would you like the anticipated link in the changest?

Reply:
Done.

Reply:
That, works. Many thanks! 🚀